ga4gh/backend.py

```diff
@@ -240,12 +252,40 @@ def filterVariantAnnotation(self, vann):
                 return False
         return True
 
+    def _checkTermEquality(self, requestedEffect, effect):
+        return "term" in requestedEffect and (
```

Comment:
Using the string "term" twice in here: (1) is this necessary (i.e., what type is requestedEffect?; (2) if it is necessary, use a variable rather than hard-coding it in twice.